retrofit/src/main/java/retrofit/DefaultConverter.java

```diff
@@ -0,0 +1,18 @@
+package retrofit;
+
+import com.squareup.okhttp.RequestBody;
+import com.squareup.okhttp.ResponseBody;
+import java.io.IOException;
+import java.lang.reflect.Type;
+
+final class DefaultConverter implements Converter {
```

Comment:
... though it still seems like that check should happen earlier. When I build a rest adapter that's when it should find the converter if it needs one, and if that's missing it should blow up.